app/services/doc_auth/socure/requests/docv_result_request.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module DocAuth
+  module Socure
+    module Requests
+      class DocvResultRequest < DocAuth::Socure::Request
+        attr_reader :document_capture_session_uuid, :biometric_comparison_required
+
+        def initialize(document_capture_session_uuid:, biometric_comparison_required: false)
+          @document_capture_session_uuid = document_capture_session_uuid
+          @biometric_comparison_required = biometric_comparison_required
+        end
+
+        private
+
+        def body
+          {
+            modules: ['documentverification'],
+            docvTransactionToken: document_capture_session.socure_docv_transaction_token,
+          }.to_json
+        end
+
+        def handle_http_response(http_response)
+          DocAuth::Socure::Responses::DocvResultResponse.new(
+            http_response: http_response,
+            biometric_comparison_required: biometric_comparison_required,
+          )
+        end
+
+        def document_capture_session
+          @document_capture_session ||=
+            DocumentCaptureSession.find_by!(uuid: document_capture_session_uuid)
+        end
+
+        def method
+          :post
+        end
+
+        def endpoint
+          @endpoint ||= URI.join(
+            IdentityConfig.store.socure_idplus_base_url,
+            '/api/3.0/EmailAuthScore',
+          ).to_s
+        end
+
+        def metric_name
+          'socure_doc_auth_docv'
```

Comment:
this API call is to the ID+ endpoint .. should it be similar to the other ID+ call we make for [kyc](https://github.com/18F/identity-idp/blob/0d726c62bb07f6a7da6eca32818f6a0857eee165/app/services/proofing/socure/id_plus/request.rb#L44)?
if so, should we distinguish between kyc and verification_data modules. `socure_id_plus_kyc` vs `socure_id_plus_verification_data`? 🤔

Reply:
May we should separate the usecases here, `resolution` and `doc_auth`, since the exact modules we call may shift over time. So:

* `socure_id_plus_doc_auth`
* `socure_id_plus_resolution`

Reply:
It becomes a bit long, but would it be clearest if it names the ID+ module, i.e. `socure_id_plus_document_verification`?

Reply:
Done in acf012960d